test/module.test.js

```diff
@@ -128,6 +128,32 @@ describe('basic', () => {
   })
 })
 
+describe('with redirect route', () => {
+  let nuxt
+
+  beforeAll(async () => {
+    const override = {
+      router: {
+        extendRoutes (routes) {
+          routes.push({ path: '/about', redirect: '/about-us' })
+        }
+      }
+    }
+
+    nuxt = (await setup(loadConfig(__dirname, 'basic', override, { merge: true }))).nuxt
+  })
+
+  afterAll(async () => {
+    await nuxt.close()
+  })
+
+  test('switchLocalePath returns redirected path', async () => {
```

Comment:
I suggest name `redirects to existing route`

The current name is not great because it `switchLocalePath` is not really relevant to the test. It's just means of testing.